test/verify/check-storage-format

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+from testlib import *
+from storagelib import *
+
+class TestStorage(StorageCase):
+    def testFormatTooSmall(self):
+        m = self.machine
+        b = self.browser
+
+        # Some day storaged/udisks will support the "dry-run-first"
+        # option.
+        #
+        # https://github.com/storaged-project/storaged/pull/82
+
+        storaged_supports_dry_run = False
```

Comment:
Yes, good point.  We can check the version of storaged.  The feature will be in 2.6.3, which hasn't been released yet.